### PR TITLE
[BACKLOG-27566] remove version from slf4j dependencies

### PR DESCRIPTION
**Pull request by Pancho7**

### CODE REVIEW

**Comment by Pancho7** [comment]
please review

File: assemblies/pmr-libraries/pom.xml [hunk 1]
```diff
@@ -1003,12 +1003,6 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
-      <version>1.7.7</version>
-    </dependency>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
```
Comment (graimundo) [inline assemblies/pmr-libraries/pom.xml]: @Pancho7 do not remove this one.

Comment (Pancho7) [reply assemblies/pmr-libraries/pom.xml]: done

**Comment by graimundo** [comment]
Replaced by: https://github.com/pentaho/big-data-plugin/pull/1640

**Pull request closed without merging**